index.js

```diff
@@ -13,6 +13,16 @@ async function generate (filename, namespace) {
     await generator.generate(`./test-data/${dataFileName}`, featureName, namespace)
 }
 
+async function generate2 (filepath, namespace) {
```

Comment:
Is this a meaningful enough name? What does it do that `generate` doesn't?

Reply:
Changed to generateFromAbsPath().